### DISCUSSION TITLE
Add RSS support

### DISCUSSION THREAD

**Discussion by Denubis** [evidence-dev/evidence#2338] [Ideas]
Looking at the blog infrastructure for evidence.dev (It'd be amazing if you could point to it being generated *by* evidence.dev), there aren't any rss feeds referenced in the page metadata. Considering your audience, I suspect not everyone is particularly inclined to give out an email.

**Comment by mcrascal** [comment]
@Denubis That's a great idea. 
Our website is an evidence project (except the docs, but we'll migrate them eventually). 

I am going to update this to track adding RSS support to evidence in general, rather than just our blog.

**Comment by jacopofar** [comment]
I think this would be very useful.
I can give it a try, but am not familiar with the stack. Is there an hook/callback to get the parsed markdown? Getting the list of files and their frontmatter annotations would make it trivial to generate the feed.

**Comment by archiewood** [comment]
You may find the files here instructive: https://github.com/evidence-dev/evidence/tree/next/packages/lib/preprocess/src/frontmatter

and you can call the pagesManifest.json API which exposes this info

https://github.com/evidence-dev/evidence/blob/next/sites/example-project/src/pages/api/pagesManifest.json/+server.js
